src/components/Schedule/Schedule.css

```diff
@@ -42,7 +42,8 @@
 	border-radius: 100px;
 	color: var(--medium-primary-color);
 	text-transform: none;
-	min-width: 16%;
+	width: 16%;
```

Comment:
Do we want varying button width? Why not full-width?

Reply:
because it is in a container with a title so I need to organize it with specifying its width

Reply:
it didn't work. I will work on it more later

Reply:
I removed it